index.html

```diff
@@ -51,6 +51,123 @@
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+                
+             </div>
+             <div class="work">
+                <img alt="work one" class="work-image" src="./assets/images/works.svg"/>
+                <h2 class="work-title">Tonic</h2>
+                <div class="work-info">
+                    <div class="work-info-title">Canopy</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">Back End Dev</div>
+                    <div class="work-info-separator"></div>
+                    <div class="work-info-others">2015</div>
+                </div>
+                <div class="work-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+                <ul class="work-stacks">
+                    <li class="stack"><a href="#" >HTML</a></li>
+                    <li class="stack"><a href="#" >Css</a></li>
+                    <li class="stack"><a href="#" >Javascript</a></li>
+                </ul>
+                <button class="work-link">See project</button>
+             </div>
+        </section>
+        <section class="about" id="about">
+            <p class="bio-header">
+               About Myself
+            </p>
+            <p class="bio-info">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="link">
+                <p class="link-text">LET'S CONNECT</p>
+                <ul class="profile-links">
+                    <li>
+                        <a href=""><img src="./assets/images/twitter.svg" alt="" srcset=""></a>
+                    </li>
+                    <li>
+                        <a href=""><img src="./assets/images/m.svg" alt="" srcset=""></a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/habitoz"><img src="./assets/images/github.svg" target="_blank"  alt="" srcset=""></a>
+                    </li>
+                    <li>
+                        <a href=""><img src="./assets/images/ok.svg" alt="" srcset=""></a>
+                    </li>
+                </ul>
+            </div>
+            <button class="work-link">Get my resume</button>
+            <div class="languages">
+                <p>Languages</p>
```

Comment:
- [ ] Kindly put the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li`